### DISCUSSION TITLE
Problems fetching remote tokens

### DISCUSSION THREAD

**Discussion by zkbdesign** [six7/figma-tokens#250] [Q&A] [answered]
Hi, 

we are a group of Designers who share a remote token source on jsonbin.io.  Each of us uses a different Master Key for the same bin as described in the documentation. Unfortunately, each time I start the plugin I get an error while fetching the source unless I add a new credential with another Master key for the same bin. Thus, I end up adding a credential for every session and so all my design co-workers. Also I get a list of providers in the sync section I cannot delete (unless I delete the credential). Is that the way it should be?

<img width="813" alt="Screenshot 2021-09-13 at 17 49 55" src="https://user-images.githubusercontent.com/89392549/133115975-77bf5a77-27af-449c-b64c-42aa1be36a52.png">

**Comment by six7** [comment]
The way sync works is you share the same id and secret for each user. You would only create the bin once and then share that with your team, give them their secret and they paste it.

I think deleting them should be possible though, I think you stumbled on a bug there, which prevents users from deleting the active credential (but as you have many different secret keys stored theres many of them).

For now you should share the secret key of the user owning the bin with the specified id with each of your team and have them add that.

And to be able to delete these credentials I'd suggest adding a new credential and activating that (in another file) and then deleting all those faulty ones.

**Comment by six7** [comment, marked as answer]
The way sync works is you share the same id and secret for each user. You would only create the bin once and then share that with your team, give them their secret and they paste it.

I think deleting them should be possible though, I think you stumbled on a bug there, which prevents users from deleting the active credential (but as you have many different secret keys stored theres many of them).

For now you should share the secret key of the user owning the bin with the specified id with each of your team and have them add that.

And to be able to delete these credentials I'd suggest adding a new credential and activating that (in another file) and then deleting all those faulty ones.
